File: java/introduction in c/4.c
//     4) The value of y is calculated as follows:-
//y = 4x3 + 2x – 6                       when x > 5
//y = 3x2 – 4y + 12                     when x < 5
//y = 6x – 5                                 when x = 5
//Write a program that accepts the value x and then computes the value of y.

#include <stdio.h>

int main() {
    int x, y;
    printf("Enter a number\n");
    scanf("%d", &x);
    if (x > 5) {
        y = 4 * x * x * x + 2 * x - 6;
        printf("The value of y is %d", y);
    } else if (x < 5) {
        y = 3 * x * x - 4 * x + 12;
        printf("The value of y is %d", y);
    } else {
        y = 6 * x - 5;
        printf("The value of y is %d", y);
    }
    return 0;
}

File: java/Code/if1/4.java
// Write a program that prompts the user for two numbers and then computes them using the following rules. If the first number is greater than the second one, the second number is subtracted from the first one. If the second number is greater than the first one, the first number is divided by the second one. Otherwise the two numbers are added.

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter two numbers: ");
        int number1 = scanner.nextInt();
        int number2 = scanner.nextInt();
        if (number1 > number2) {
            System.out.println("Result: " + (number1 - number2));
        } else if (number2 > number1) {
            System.out.println("Result: " + (number1 / number2));
        } else {
            System.out.println("Result: " + (number1 + number2));
        }
    }
}
